tests/System/expected/test_webContext__TagManager.getAvailableVariableTypesInContext.xml

```diff
@@ -1005,6 +1005,40 @@
 						<condition>!requireConsent &amp;&amp; !disableCookies</condition>
 						<fullWidth>0</fullWidth>
 					</row>
+					<row>
+						<name>customCookieTimeOutEnable</name>
+						<title>Enable custom visitor Cookie timeout</title>
+						<value>0</value>
+						<defaultValue>0</defaultValue>
+						<type>boolean</type>
+						<uiControl>checkbox</uiControl>
+						<uiControlAttributes>
+						</uiControlAttributes>
+						<availableValues />
+						<description>Allows you to set a custom time in seconds for the _pk_id cookie</description>
+						<inlineHelp />
+						<templateFile />
+						<introduction />
+						<condition>!disableCookies</condition>
+						<fullWidth>0</fullWidth>
+					</row>
+					<row>
+						<name>customCookieTimeOut</name>
+						<title>The pk_id expire time in seconds</title>
+						<value>33955200</value>
+						<defaultValue>33955200</defaultValue>
+						<type>int</type>
```

Comment:
Looks like the failing test is expecting `integer` instead of `int`. Please update.